scaladoc/src/dotty/tools/scaladoc/tasty/comments/MemberLookup.scala

```diff
@@ -191,8 +201,18 @@ trait MemberLookup {
             else if s.flags.is(Flags.Module) then tm = Some(s)
             else if s.isClassDef || s.isTypeDef then tp = Some(s)
           }
+
+          def downwardLookUpForInheritedMembers(qs: List[String], owner: Symbol): Option[(Symbol, Option[Symbol])] =
+            downwardLookup(qs, owner).orElse(owner.memberMethod(qs.head).headOption.map((_, Some(owner)))).orElse {
+              val symbol = owner.memberType(qs.head)
+              Option.when(symbol != dotty.tools.dotc.core.Symbols.NoSymbol)(symbol).map((_, Some(owner)))
+            }.orElse {
+              val symbol = owner.memberField(qs.head)
+              Option.when(symbol != dotty.tools.dotc.core.Symbols.NoSymbol)(symbol).map((_, Some(owner)))
+            }
```

Comment:
Ah, it seems that the code we had previously didn't actually look up inherited members at all? In that case it'll have to be changed. We don't want to simply go through Scala Reflect functions here, we also need to take into account absent symbols (see `hackIsNotAbsent`).

This code also doesn't seem to work when the inherited symbol is in the middle of the query. In general it'd be better to modify `localLookup` instead of `downwardLookup`.